src/main.cpp

```diff
@@ -2,62 +2,358 @@
+    } else {
+        PromptedOutputStream pos = PromptedOutputStream(values.path);
+        pos.Stream().write((char *)builder.Buffer(), builder.BufferLength());
+    }
+
 }
 
-static void PipelineRun() {
-    PipelineInputList input = PromptPipelineInput();
-    Pipeline pipeline = PromptPipeline();
+static void PipelineRun(const PipelineOptions &values) {
+    PipelineInputList input = GetPipelineInput(values);
+    Pipeline pipeline = SetPipeline(values);
     std::vector<PipelineOutput> outputs = pipeline.Go(input);
-    PromptPipelineComparison(input, outputs);
+    PipelineComparison(input, outputs, values);
 }
 
-static void PipelineBenchmark() {
-    PipelineInputList input = PromptPipelineInput();
-    Pipeline pipeline = PromptPipeline();
-    int iterations = Prompt<int>("Times to run the pipeline");
-    std::cerr << "Benchmarking..." << std::endl;
-
-    // TODO: we can do better than this :| maybe include mean time, 99% time, or allow a vector of
-    // input and determine which one took the longest
-    auto startTime = std::chrono::high_resolution_clock::now();
-    for (int i = 0; i < iterations; i++) {
-        pipeline.Go(input);
-    }
-    auto endTime = std::chrono::high_resolution_clock::now();
-    auto totalTime = std::chrono::duration<double, std::milli>(endTime - startTime);
-    std::cout << "total_ms " << totalTime.count() << std::endl;
-}
+// DO NOT DELETE
+// static void PipelineBenchmark() {
+//     PipelineInputList input = PromptPipelineInput();
+//     Pipeline pipeline = PromptPipeline();
+//     int iterations = Prompt<int>("Times to run the pipeline");
+//     std::cerr << "Benchmarking..." << std::endl;
+
+//     // TODO: we can do better than this :| maybe include mean time, 99% time, or allow a vector of
+//     // input and determine which one took the longest
+//     auto startTime = std::chrono::high_resolution_clock::now();
+//     for (int i = 0; i < iterations; i++) {
+//         pipeline.Go(input);
+//     }
+//     auto endTime = std::chrono::high_resolution_clock::now();
+//     auto totalTime = std::chrono::duration<double, std::milli>(endTime - startTime);
+//     std::cout << "total_ms " << totalTime.count() << std::endl;
+// }
 
 }
 
 int main(int argc, char **argv) {
-    lost::RegisterCliArgs(argc, argv);
-    std::cerr << "LOST: Open-source Star Tracker" << std::endl;
-    lost::InteractiveChoice<void (*)()> mainChoices;
-    mainChoices.Register("pipeline", "Run a pipeline", &lost::PipelineRun);
-    mainChoices.Register("benchmark", "Benchmark a pipeline", &lost::PipelineBenchmark);
-    mainChoices.Register("build_database", "Build database from catalog", &lost::DatabaseBuild);
-    (*mainChoices.Prompt("Choose action"))();
-}
+
+    if (argc == 1) {
+        std::cout << "Usage: ./lost database or ./lost pipeline" << std::endl << "Use --help flag on those commands for further help" << std::endl; 
```

Comment:
Also print this message when the user types an invalid first argument, eg `./lost boop`